### ISSUE TITLE
GBMV/ Personalmodul Sonderjobs

### ISSUE THREAD

**Issue by Toelk-A** [chrbusch/GTM-V#613]
trage ich den Sonderjob Urlaubswunsch vom 30.12.2019 - 03.01.2020 ein, so wird noch immer pro Monat gesplittet. Man kann noch immer keinen Urlaub über den Monatswechsel/ Jahreswechsel hinaus in einem Schritt dokumentieren. Bitte erneut prüfen

**Comment by chrbusch** [comment]
Ich denke, es sollte genau so sein, wie es ist; Sie haben ja auch JAHRESurlaub, gegen den wir prüfen müssen.
Daher braucht das System Urlaube, die ganz in ein Kalenderjahr fallen

**Comment by Toelk-A** [comment]
es funktioniert auch nicht bei Monatswechseln im selben Jahr...

**Comment by chrbusch** [comment]
Ja, Sie haben Recht - aber nur, wenn der Folgemonat der Januar ist…
Trotzdem BUGFIX, keine Frage.
Bitte nun erneut testen
